scripts/tests/chiptest/__init__.py

```diff
@@ -138,7 +138,10 @@ def _GetInDevelopmentTests() -> Set[str]:
         "TestEqualities.yaml",              # chip-repl does not support pseudo-cluster commands that return a value
         "TestExampleCluster.yaml",          # chip-repl does not load custom pseudo clusters
         "TestClientMonitoringCluster.yaml",  # Client Monitoring Tests need a rework after the XML update
-        "Test_TC_TIMESYNC_1_1.yaml"         # Time sync SDK is not yet ready
+        "Test_TC_TIMESYNC_1_1.yaml",         # Time sync SDK is not yet ready
```

Comment:
we should probably update the comment about "Currently this is empty" to something different as this is obviously not empty.

I wonder if we should add some "date added" in comments to see how long these yamls have been in development and have some idea of how to prioritize things.

Reply:
I have removed the part of the comment related to emptiness and I have added the date as you suggest.